src/Auth0Client.ts

```diff
@@ -1194,7 +1204,8 @@ export default class Auth0Client {
           redirect_uri,
           ...(timeout && { timeout }),
           auth0Client: this.options.auth0Client,
-          useFormData: this.options.useFormData
+          useFormData: this.options.useFormData,
+          timeout: this.httpTimeoutMs
```

Comment:
You're overriding the `timeout` in 1205

Not sure if anyone is doing `getTokenSilently({ timeout: ...  })` since it's not documented, but we should probably keep that working

Reply:
Good spot, was just an oversight.

I think it should still resolve to the same thing but agree it's a bit of a smell.